projects/cm_mcu/MonitorI2CTask.h

```diff
@@ -0,0 +1,81 @@
+/*
+ * MonitorI2CTask.h
+ *
+ *  Created on: June 30, 2022
+ *      Author: pkotamnives
+ */
+
+#ifndef PROJECTS_CM_MCU_MONITORI2CTASK_H_
+#define PROJECTS_CM_MCU_MONITORI2CTASK_H_
+
+#include "common/smbus.h"
+#include "semphr.h"
+#include "Tasks.h"
+
+struct sm_command_t {
+  int reg_size;          // number of bytes of register/command
+  unsigned char page;    // I2C page address
+  unsigned char command; // I2c register address
+  int size;              // number of bytes to read
+  char *name;            // text describing command
+  uint16_t bit_mask;     // begin bit mask
+  char *units;           // units for pretty printing
+  enum pm_type type;     // how to decode command (L11 or bitfield or ...)
+};
+
+// how to find an I2C device, with a mux infront of it.
+
+struct MonitorI2CTaskArgs_t {
+  const char *name;                  // name to be assigned to the task
+  struct dev_moni2c_addr_t *devices; // list of devices to query
+  int i2c_dev;                       // i2c controller #
+  int n_devices;                     // number of devices
+  struct sm_command_t *commands;     // list of commands
+  const uint8_t n_commands;          // number of commands
+  const int n_values;                // number of results
+  const uint8_t n_pages;             // number of pages to loop over
+  uint16_t *sm_values;
+  tSMBus *smbus;                       // pointer to I2C controller
+  volatile tSMBusStatus *smbus_status; // pointer to I2C status
+  TickType_t updateTick;               // last update time, in ticks
+  SemaphoreHandle_t xSem;              // semaphore for controlling access to device
+  bool requirePower;                   // true if device requires power
```

Comment:
get rid of this here (see the other comment in the related .c file)

Reply:
Done.